src/printer.js

```diff
@@ -46,6 +46,17 @@ function genericPrint(path, options, printPath) {
     return linesWithoutParens;
   }
 
+  // Escape hatch
+  if (node.comments &&
+      node.comments.length > 0 &&
+      node.comments[0].value.trim() === 'prettier-ignore-next' &&
+      node.comments[0].leading) {
+    return options.originalText.slice(
+      util.locStart(node),
+      util.locEnd(node)
+    );
+  }
```

Comment:
Try adding some test cases where the expression is an argument to function, a property of an object, etc. Those interactions are where things might get weird.

The problem is the above code will naively output `\n`'s and will mess up the measurement system. We might need split the content on newline and insert our own hardlines to make sure it never gets messed up. We might should add a `breakParent` just to make sure it always breaks up the expression too.